pkgs/applications/editors/emacs/elisp-packages/manual-packages/typst-preview/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  melpaBuild,
+  websocket,
+  f,
+  fetchFromGitHub,
+}:
+melpaBuild {
+  pname = "typst-preview";
+  version = "0-unstable-2025-10-13";
+  src = fetchFromGitHub {
+    owner = "havarddj";
+    repo = "typst-preview.el";
+    rev = "74244aef2545d7bb79b5396dc6503e23e6d239a7";
+    hash = "sha256-Bt2YvUpHx5vR+NwUVfk21h0BrLbuW182E0qqx1DNIOo=";
+  };
+
+  packageRequires = [
+    websocket
+    f
+  ];
+
+  meta = {
+    description = "Typst live preview minor mode for emacs";
+    homepage = "https://github.com/havarddj/typst-preview.el";
+    license = lib.licenses.unlicense;
```

Comment:
Isn't it [`gpl3Plus`][1]?

[1]: https://github.com/havarddj/typst-preview.el/blob/74244aef2545d7bb79b5396dc6503e23e6d239a7/typst-preview.el#L13-L16